contents/tutorials/bounce-rate.md

```diff
@@ -0,0 +1,57 @@
+---
+title:  How to calculate bounce rate
+date: 2023-06-21
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-15.png
+tags: ['insights', 'hogql']
+---
+
+Bounce rate is the percentage of users who leave your page immediately after visiting. It is a popular marketing metric showing the relevance and engagement of content for site visitors.  
+
+This tutorial shows you how to calculate bounce rate in PostHog. To get started, you need to install the [snippet](/docs/getting-started/install?tab=snippet) or [JavaScript SDK](/docs/libraries/js) and enable "record user sessions"  in [project settings](https://app.posthog.com/project/settings).
+
+## Calculating bounce rate with SQL insights
+
+To calculate bounce rate, we need data from `raw_session_replay_events`, which we can access with [SQL insights](/docs/product-analytics/hogql#sql-insights). To create a new SQL insight, go to the insight tab, click [new insight](https://app.posthog.com/insights/new), then go to the SQL tab. This is where we write our SQL statement.
+
+We count a bounce as a session where the user is active for less than 3 seconds. To do this in SQL, we get a count of sessions (using `session_id`) where `active_milliseconds` is less than `3000` and divide by the total session count, then multiply by `100`. Together, this looks like this:
+
+```sql
+select (
+	count(
+		multiIf(active_milliseconds < 3000, session_id, NULL)
+	) / count(session_id)
+) * 100 as bounce_rate
+from raw_session_replay_events
+```
+
+This gives us a bounce rate percentage insight we can save, update, and add to dashboards.
+
+![Bounce rate](../images/tutorials/bounce-rate/bounce-rate.mp4)
+
+> **Why not use single page visits divided by total visits?** The `raw_session_replay_events` data does not include the frequency of events and the `events` data does not include session details, making calculating sessions with a single `$pageview` event not possible (at the moment).
+
+## Using different bounce criteria
+
+Although we used active time as our criteria for bounce rate, PostHog has other options. They include using `click_count`, `keypress_count`, or `mouse_activity_count`. We can find these  in the [database data management](https://app.posthog.com/data-management/database) tab under the `raw_session_replay_events` table.
+
+> **How does Google Analytics 4 calculate the bounce rate?** It is the percentage of sessions that **do not** last longer than 10 seconds, have a conversion event, or at least 2 pageviews or screenviews.
```

Comment:
We can hyperlink to GA4 doc here -> https://support.google.com/analytics/answer/12195621?hl=en

Perhaps we should use this definition in our tutorial, or maybe show how to calculate a variety of different definitions?

Reply:
Agreed. Most people who want to track this will be familiar with GA.

Reply:
Added